api/src/main/resources/db/changelog/db.changelog-fees-managed-v5.yaml

```diff
@@ -150,3 +150,16 @@ databaseChangeLog:
           sql: >
             INSERT INTO service_type (name,creation_time,last_updated)
             VALUES ('other',current_timestamp,current_timestamp) ON CONFLICT DO NOTHING
+
+- changeSet:
+    id: 1530721907440
+    author: sachi
+    preConditions:
+      - onFail: MARK_RAN
+      - dbms:
+          type: postgresql
```

Comment:
@jaleen  - may be we should consider database containers for tests. https://www.testcontainers.org/usage/database_containers.html